js/dom.js

```diff
@@ -27,6 +27,8 @@ Object.assign(EventTarget.prototype, {
 
 //#region Exports
 
+const $root = document.documentElement;
```

Comment:
IMO, it doesn't worth adding a global dependency just to type less. Though it should be fine if you want to keep it.

Reply:
Well, DOM is so annoyingly verbose that the code becomes unreadable... But indeed, adding a global isn't worth it. I've switched to `$.root` and `$.rootCL`. I guess the same can be done later for `$create` and `$remove`